content/analytics/data-views/data-view-data-type-conversion.md

```diff
@@ -0,0 +1,96 @@
+---
+uid: data-view-data-type-conversion
+---
+
+# Data view data type conversion
+
+When requesting a data view that includes multiple data types in a single column, the data view will:
```

Comment:
This is like the intro to a bulleted list, but "the data view will..." doesn't go with step 2. I would change to

"When you request a data view that includes multiple data types in a single column:
1. The data view converts ..."